eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,49 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo ($serviceDirectoryName="*")
```

Comment:
Why `*` as opposed to empty?

Reply:
So am passing that down to python scrip, land if no service directory is specified than it will be `sdk/*/*/setup.py` as opposed to `sdk/service_directory/*/setup.py` when a service directory is specified.